### ISSUE TITLE
PBEh-3c: Disagreement between Psi4 and ORCA Interaction Energies

### ISSUE THREAD

**Issue by carolinesargent** [psi4/psi4#2304]
Hi all,

I have been benchmarking energy calculations of various organic small-molecule dimers in Psi4. These dimers have minimum monomer separations (closest intermolecular contact) up to 15 Å. To determine which method produces interaction energies closest to coupled cluster results, I have graphed the partial crystal lattice energy contributions (a scaled sum of the dimer interaction energies) for subsets of dimers with monomer separations between (n, n+1).  Odd behavior arose when analyzing results of benzene. PBEh-3c returned errors _much_ larger than any other method. To investigate these results, PBE, PBEh, and PBEh-3c were run in ORCA (5.0.1) and Psi4 (1.3rc3). Results are shown below. Methods denoted with an asterisk are those which use a fine grid (total points = 6340328). Methods without an asterisk used the default grid (Psi4 total points = 253270; ORCA total points = 56510).

![subplot-8](https://user-images.githubusercontent.com/87093097/134976005-f5bf7501-27d2-40c0-b033-06b33126f8dc.png)

The graph shows that while PBE generally agrees in Psi4 and ORCA, PBEh and PBEh-3c disagree when comparing the two software packages and default grid points are used. When Psi4 uses a fine grid, the results align more-so with ORCA -- and the coupled cluster results.

Two individual dimers have been chosen to highlight the differences in interaction energies:

| Benzene Dimer | Min. Monomer Separation (Å) | Interaction Energy (kJ/mol) | Software | Spherical Points | Radial Points | Total Points |
|---|---|---|---|---|---|---|
| 2mer-0+1 | 2.544 | -6.01017169 | ORCA | 194 | OptM3 with GC (2021) | 56510 |
| | | -6.09347943 | Psi4 | 302 | 75 | 253270 |
| | | -6.06158176 | Psi4 | 5810 | 99 | 6340328 |
| 2mer-0+77 | 11.016 | -0.03777130 | ORCA | 194 | OptM3 with GC (2021) | 56510 |
| | | -0.09279455 | Psi4 | 302 | 75 | 253270 |
| | | -0.03957357 | Psi4 | 5810 | 99 | 6340328 |

To me, this seems to be a systematic issue with Psi4 calculating interaction energies with PBEh / PBEh-3c. Does anyone know of a reason why this might be occurring? 

Thanks y'all!

**Comment by loriab** [comment]
Thanks!

* this is dimer - m1 - m2, and the monomer geometries are uniform, so what you're seeing is a big increase in inaccuracy for the dimer depending on if its extent is near or far. is that right?
* any idea about "OptM3 with GC" characteristics and how it relates to defgrid2 https://sites.google.com/site/orcainputlibrary/numerical-precision?

**Comment by carolinesargent** [comment]
Yes -- that is right. It may also be important to note that the monomer m2 may not be in the exact same orientation relative to m1 for both dimers. 

I am having trouble finding any information about ORCA's "OptM3 with GC," so I'm not sure how it relates to defgrid2 (yet). The ORCA output files simply state:
`Radial Grid Type               RadialGrid  . . .  OptM3 with GC (2021)`

**Comment by hokru** [comment]
OptM3 with GC is the Treutler-Ahlrichs M3 mapping with Gauss-Chebyschev (GC) quadrature. This is similar to what psi4 does.
However, the radial points in psi4 are fixed numbers while in ORCA they vary for each element and were optimized against some data set (hence `OptM3 with GC`)

For benchmarking interaction energies one wants `defgrid3` in ORCA and ``590/99`` in psi4. 

Changing the monomer orientation will also have some influence. Though ORCA has an optional feature that makes grid rotational invariant but I dont know how it works.
